marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -254,26 +254,26 @@ public boolean isServerVersionValid(String versionString) {
                 return false;
             }
             if(serverVersion.isNightly()){
-                //Support all 10.0-nightly on or after 6/20/2019 and 9.0-nightly on or after 7/25/2019
+                //Support all 10.0-nightly on or after 9/12/2019 and 9.0-nightly on or after 11/12/2019
```

Comment:
Why are these dates important? This comment is just stating the same thing as the code - the comment should instead address what the code cannot, which is why 9/12 and 11/12 are the dates.

Reply:
These dates are the nightly servers on the day the server was released (looking into builds dir). I will add a comment to that effect.